utils/nctl/templates/chainspec.toml

```diff
@@ -29,6 +29,15 @@ auction_delay = 3
 # The delay for the payout of funds, in eras. If a withdraw request is included in a block in era N
 # (other than the last one), they are paid out in the last block of era N + locked_funds_period.
 locked_funds_period = 15
+# Round seigniorage rate represented as a fractional number
```

Comment:
Maybe a bit clearer: "…as a fraction of the total supply"
(At first I wasn't sure whether it's in motes.)